azurerm/internal/services/iottimeseriesinsights/tests/resource_iot_time_series_insights_gen2_environment_test.go

```diff
@@ -0,0 +1,238 @@
+package tests
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/clients"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/iottimeseriesinsights/parse"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func TestAccAzureRMIoTTimeSeriesInsightsGen2Environment_basic(t *testing.T) {
```

Comment:
Should we use the test shim for the tests?

Reply:
From talking internally, we're holding off on this at the moment